src/plugins/select-field/select-field-display.js

```diff
@@ -46,7 +47,8 @@ export default function SelectFieldDisplay({ content, input, canModifyInput, onI
                 key={item.key}
                 value={item.key}
                 checked={selectedValues.includes(item.key)}
-                onChange={canModifyInput ? handleSelectionChange : null}
```

Comment:
`onChange={handleSelectionChange}` has to stay (without the ternary as in disabled state it is not called anyways). This will also fix the linting error.

Reply:
Yeah I wanted to just remove the ternary but my hand was too fast for my brain 😮‍💨 Thanks, corrected.